framer/Context.coffee

```diff
@@ -254,3 +260,44 @@ class exports.Context extends BaseClass
 	@define "size",  get: -> _.pluck(@frame, ["x", "y"])
 	@define "point", get: -> _.pluck(@frame, ["width", "height"])
 
+	@define "backgroundColor",
+		get: ->
+			return @_element?.style["backgroundColor"]
+		set: (value) ->
+			if Color.isColor(value)
+				@_element?.style["backgroundColor"] = new Color value.toString()
+
+	@define "perspective",
+		get: ->
+			return @_perspective
+		set: (value) ->
+			if _.isNumber(value)
+				@_perspective = value
+				@_element?.style["perspective"] = @_perspective
+				@_element?.style["webkitPerspective"] = @_perspective
```

Comment:
Double? We don't do that anywhere.

If we're going to do css prefixing this would be my least favorite way :-)